.github/workflows/pr_tests.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       if: ${{ always() }}
       uses: actions/upload-artifact@v4
       with:
-        name: pr_${{ matrix.config.report }}_test_reports
+        name: pr_${{ matrix.config.framework }}_${{ matrix.config.report }}_test_reports
```

Comment:
To preserve uniqueness.

Reply:
this one seems to be ok - config.report already named nicely with framework taken into account